client/components/ConfigureActiveRuns/index.jsx

```diff
@@ -203,7 +341,34 @@ class ConfigureActiveRuns extends Component {
             at_browser_pairs: atBrowserPairs
         };
 
-        dispatch(saveRunConfiguration(config));
+        const configurationChanges = this.setConfigChanges(config);
+
+        if (configurationChanges.length > 0) {
+            this.setState(
+                {
+                    configurationChanges,
+                    newConfiguration: config
+                },
+                () => {
+                    this.showChanges();
+                }
+            );
+        } else {
```

Comment:
If there are no changes, we should have a popup that says there are no changes -- rather than just try to save, right? I feel a bit strange about this else here.

I think we should only save if you click the confirmation on the modal.